packages/ts-transformers/src/internal/decorators/property.ts

```diff
@@ -43,7 +43,8 @@ export class PropertyVisitor implements MemberDecoratorVisitor {
     property: ts.ClassElement,
```

Comment:
Nit: Maybe worth renaming since this node can be both a property and a get accessor declaration node?